my-ideas

```diff
@@ -0,0 +1,11 @@
+<AARMN>
+
+1-I think it's better to add more widgets and layouts to it
+2-Kivy have a problem in right click that put red dots I wish in next version this ability be off as default
+3-its designer project is unpleasent to use for RAD designers I wish for beta or stable versions
+4-incorrect position of touch in my linux device (you should touch upper than your widget to touch that widget
```

Comment:
Please open a bug report at the Kivy repo, describing the issue in detail.
https://github.com/kivy/kivy